hicommonwealth/commonwealth
# Improving sidebar navigation on an unscoped page

## Link to Issue
Closes: #7987 

## Description of Changes
- Adds enhanced sidebar information display.
- Displays user profile information including:
  - Username.
  - Profile Image.
  - Background Image.
  - Linked addresses.

## "How We Fixed It"

## Test Plan
- Open the side bar.
- Verify that it display the user profile and background image.
- Verify that it display the connected addresses on the sidebar.
- Remove the address and verify that you can also remove linked address from the sidebar.

## Deployment Plan

1. 

## Other Considerations

-
